src/dealer/OkexPerpetualSwapStrategy.ts

```diff
@@ -58,31 +59,31 @@ export class OkexPerpetualSwapStrategy implements IHedgingStrategy {
         lastBtcPriceInUsd,
         hedgingBounds,
       )
-      logger.debug(
-        `getHedgingOrderIfNeeded(${{
-          liabilityInUsd,
-          exposureInUsd,
-          lastBtcPriceInUsd,
-          hedgingBounds,
-        }}) returned: ${hedgingOrder}`,
-      )
+      logger.debug({
+        called: `getHedgingOrderIfNeeded(${liabilityInUsd}, ${exposureInUsd}, ${lastBtcPriceInUsd}, hedgingBounds)`,
+        hedgingBounds: hedgingBounds,
+        returned: hedgingOrder,
+      })
 
       if (hedgingOrder.output.tradeSide === TradeSide.NoTrade) {
         const msg = `${hedgingOrder.input.loBracket} < ${hedgingOrder.input.exposureRatio} < ${hedgingOrder.input.hiBracket}`
         logger.debug(`Calculated no hedging is needed: ${msg}`)
       } else {
         if (hedgingOrder.output.IsSimulation) {
-          logger.debug(`Calculated a SIMULATED new hedging order: ${hedgingOrder}`)
+          logger.debug({
+            msg: "Calculated a SIMULATED new hedging order",
+            order: hedgingOrder,
+          })
         } else {
-          logger.debug(`Calculated a new hedging order: ${hedgingOrder}`)
+          logger.debug({ msg: "Calculated a new hedging order", order: hedgingOrder })
```

Comment:
this is not how the logger should be used. logger documentation can be read here: https://github.com/pinojs/pino/

typically you have `log.info({ variables }, "your logging message without any variable")